docs/site/Booting-an-Application.md

```diff
@@ -237,6 +237,33 @@ The `datasources` object support the following options:
 | `nested`     | `boolean`            | `true`               | Look in nested directories in `dirs` for DataSource artifacts                                                 |
 | `glob`       | `string`             |                      | A `glob` pattern string. This takes precendence over above 3 options (which are used to make a glob pattern). |
 
+### Service Booter
+
+#### Description
+
+Discovers and binds Service providers using `app.serviceProvider()` (Application
+must use `ServiceMixin` from `@loopback/service-proxy`).
+
+**IMPORTANT:** For a class to be recognized by `ServiceBooter` as a service
+provider, its name must end with `Provider` suffix and its prototype must have a
+`value()` method.
```

Comment:
To me, this is a sign that we should have something like https://github.com/strongloop/loopback-next/pull/992 to make it possible to explicit.

Reply:
+1 for #992, it's certainly needed.